src/main/java/seedu/todo/logic/commands/AddCommand.java

```diff
@@ -61,6 +63,8 @@ public CommandResult execute() throws ValidationException {
         if(!model.getObservableList().contains(addedTask)) {
             model.view(TaskViewFilter.DEFAULT);
         }
+        eventBus.post(new HighlightTaskEvent(addedTask));
+        eventBus.post(new ExpandCollapseTaskEvent(addedTask));
```

Comment:
Since this is a toggle event, if the event is expanded, it will get collapsed, right?